web_hosting_manager/app/components/app_logs.js

```diff
@@ -14,27 +14,27 @@ export default class AppLogs extends Component {
     };
   }
   componentDidMount() {
-    this.setState({loading: true});
-    this.props.getLogFilePath()
-      .then(path => {
+    this.setState({ loading: true }); // eslint-disable-line react/no-did-mount-set-state
+    this.props.getLogFilePath() // eslint-disable-line react/prop-types
```

Comment:
let's add this to a `propTypes` declaration for clearer errors etc, and then the linter will be happy too.